cosim/black-parrot-example/vivado/vivado-create-block.pynqz2.2019.1.tcl

```diff
@@ -87,12 +87,19 @@ delete_bd_objs [get_bd_nets reset_rtl_0_1] [get_bd_ports reset_rtl_0]
 connect_bd_net [get_bd_pins processing_system7_0/FCLK_RESET0_N] [get_bd_pins proc_sys_reset_0/ext_reset_in]
 save_bd_design
 
-# change this to a 0 to have it stop before synthesis and implementation
+# Change to 0 to have it stop before synthesis / implementation
 # so you can inspect the design with the GUI
 
 if {1} {
-launch_runs impl_1 -to_step write_bitstream -jobs 4
-wait_on_run impl_1
+  launch_runs synth_1 -jobs 4
+  wait_on_run synth_1
+  open_run synth_1 -name synth_1
```

Comment:
does this open vivado?

Reply:
This will not open GUI

Reply:
what does it do? (sorry, I don't know vivado well)

Reply:
I think it just opens the synthesized project but without GUI. We need to open it before we can apply timing constraints to it

Reply:
kk!